app/components/forms/bill-of-lading/component.js

```diff
@@ -1,23 +1,25 @@
-import Ember from 'ember';
-import computed from 'ember-computed-decorators';
-import moment from 'moment';
+import Ember from "ember";
+import computed from "ember-computed-decorators";
+import moment from "moment";
 
 export default Ember.Component.extend({
-  classNames: ['col', 'stretch', 'card-1'],
+  classNames: ["col", "stretch", "card-1"],
 
-  @computed('model.orderItems')
+  @computed("model.orderItems")
   validOrderitems(orderItems) {
     return orderItems
-      .filter(orderItem => orderItem.get('quantity') > 0);
+      .filter(orderItem => orderItem.get("quantity") > 0);
   },
 
-  @computed('validOrderitems.@each.{quantity}')
-  totalQuantity(orderItems) {
-    return orderItems.reduce((acc, cur) => acc + cur.get('quantity'), 0);
+  @computed("validOrderitems.@each.{total}")
```

Comment:
I think we should decouple the model from the row. Instead of passing a complex object in, let's pass:

1. name
1. unit_price
1. quantity

Then this can be reverted to the original.

This will help reuse later.

For styling the total row, we should do that outside of the row styles. Will be easier to add a `total` class on that row and then style it at the outer component level.

So the new api should be something like:

```hbs
{{rows/credit-note-row
   name=cni.item.name
   quantity=cni.quantity
   unit_price=cni.quantity}}

{{rows/credit-note-row
   class="total"
   name=cni.item.name
   quantity=cni.quantity
   unit_price=cni.quantity}}
```